src/main/java/com/google/cloud/tools/gradle/endpoints/framework/server/EndpointsServerPlugin.java

```diff
@@ -75,59 +79,83 @@ public void execute(Project project) {
         project.getArtifacts().add(ARTIFACT_CONFIGURATION, discoveryDocArchive);
       }
     });
-
   }
 
   private void createGenerateDiscoveryDocsTask() {
     project.getTasks().create(GENERATE_DISCOVERY_DOC_TASK, GenerateDiscoveryDocsTask.class,
-        new Action<GenerateDiscoveryDocsTask>() {
-          @Override
-          public void execute(final GenerateDiscoveryDocsTask genDiscoveryDocs) {
-            genDiscoveryDocs.setDescription("Generate endpoints discovery documents");
-            genDiscoveryDocs.setGroup(APP_ENGINE_ENDPOINTS);
-            genDiscoveryDocs.dependsOn(JavaPlugin.CLASSES_TASK_NAME);
-
-            project.afterEvaluate(new Action<Project>() {
-              @Override
-              public void execute(Project project) {
-                File classesDir = project.getConvention().getPlugin(JavaPluginConvention.class)
-                    .getSourceSets().getByName(SourceSet.MAIN_SOURCE_SET_NAME).getOutput()
-                    .getClassesDir();
-                genDiscoveryDocs.setClassesDir(classesDir);
-                genDiscoveryDocs.setDiscoveryDocDir(extension.getDiscoveryDocDir());
-                genDiscoveryDocs.setServiceClasses(extension.getServiceClasses());
-                genDiscoveryDocs.setWebAppDir(
-                    project.getConvention().getPlugin(WarPluginConvention.class).getWebAppDir());
-              }
-            });
-          }
-        });
+      new Action<GenerateDiscoveryDocsTask>() {
```

Comment:
please undo the spacing changes here, they were correct before (offset by 2x for line continuation)